docs/userguides/accounts.md

```diff
@@ -66,6 +66,16 @@ ape accounts import <ALIAS>
 It will prompt you for the private key.
 If you need help exporting your private key from Metamask, see [this guide](https://metamask.zendesk.com/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
 
+You can also import accounts from mnemonic seed by using the --mnemonic flag:
+
+```bash
+ape accounts import --mnemonic <ALIAS>
+```
+
+It will prompt you for the [mnemonic seed](https://metamask.zendesk.com/hc/en-us/articles/360015290032-How-to-reveal-your-Secret-Recovery-Phrase). 
```

Comment:
The link is very specific to meta mask and not a general concept of a mnemonic seed. Do we want to find another resource for a mnemonic seed? If we want something specific for meta mask, we could change the linking text to represent it as a meta mask resource.

Reply:
I can find a better link, my thought process using metamask docs was only because they were used already on line 67 previously.

Reply:
Makes total sense, I think the added context on the previous link lets a user know it's specific to Metamask. Maybe we can update what you already have to give that context as well, something like:
```suggestion
It will prompt you for the mnemonic seed. If you need help finding your mnemonic seed (Secret Recovery Phrase) in Metamask, see [this guide](https://metamask.zendesk.com/hc/en-us/articles/360015290032-How-to-reveal-your-Secret-Recovery-Phrase). 
```

Reply:
> Makes total sense, I think the added context on the previous link lets a user know it's specific to Metamask. Maybe we can update what you already have to give that context as well, something like:

I think this is a good idea, but maybe the words `mnemonic seed` should also link to an explanation somewhere of what the mnemonic seed is.